inputWidget/1.html

```diff
@@ -5,8 +5,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Input tag</title>
+    <link rel="stylesheet" href="1.css">
 </head>
 <body>
-    
+    <div class="parent">
+        <div class="num">Enter a number between 1 to 100</div>
+        <input type="text" class="inpElement" value="" />
```

Comment:
use input-type as numbers, using text as input type would allow the user to type any alphanumeric characters which is not the use case/